Replace node+npm+webpack by esbuild (#4377)
phoenixframework/phoenix: guides/deployment/deployment.md

## Compiling your application assets

This step is required only if you have compilable assets like JavaScript and stylesheets. By default, Phoenix uses `esbuild` but everything is encapsulated in a single `mix assets.deploy` task defined in your `mix.exs`:

```console
$ MIX_ENV=prod mix assets.deploy
Check your digested files at "priv/static".
```

> And that is it! The first command builds the assets and the second generates digests as well as a cache manifest file so Phoenix can quickly serve assets in production.

Reviewer: Now it's just a simple command, make sense to change it too?

> And that is it! The first command builds the assets and the second generates digests as well as a cache manifest file so Phoenix can quickly serve assets in production.

Author: ```suggestion
And that is it! The Mix task by default builds the assets and then generates digests with a cache manifest file so Phoenix can quickly serve assets in production.
```
